src/components/common/StarredItemIcon/StarredItemIcon.tsx

```diff
@@ -21,8 +21,8 @@ const StarOutline = styled(StarBorderOutlined)(({ theme }) => ({
 
 export const StarredItemIcon: React.FC<Props> = ({ isStarred }) => {
   return isStarred ? (
-    <StarFilled titleAccess="Filled Star Icon" />
+    <StarFilled titleAccess="Star" data-testid="Filled Star Icon" />
```

Comment:
Is the `titleAccess` for accessibility? If so, I wonder if these should still be Filled Start/Outline Star?

Reply:
The `titleAccess` value shows on the screen as a title when you hover over the icon.

Reply:
Got it. So should they say "Star" when it is not starred and "Unstar" when it is starred? Since I think that all of these stars are buttons that toggle the starred state.

Reply:
Ah yeah good idea